pom.xml

```diff
@@ -556,24 +557,43 @@
 			</dependency>
 			<!-- Testing: JUnit -->
 			<dependency>
-				<groupId>junit</groupId>
-				<artifactId>junit</artifactId>
-				<version>4.13.1</version>
+				<groupId>org.junit.jupiter</groupId>
+				<artifactId>junit-jupiter-engine</artifactId>
+				<version>${junit.version}</version>
+				<scope>test</scope>
```

Comment:
Can you remove the scope declaration here (and for the other new dependencies)? We've recently stepped away from declaring scope in the dependency management section of the root pom, because it may have some unexpected effects in downstream projects (see GH-2812). 

In this case this is of course very unlikely to ever be a problem (I doubt many projects would use a scope other than test for junit), but let's stick to the principle: dependencyManagement declares only versions, scope is declared where the actual dependency is needed.

Reply:
I think I can remove the scope, but that would make it the default scope compile wouldn't it?
Anyway the compliance suites still use the original junit4 approach and in compile scope if I figured that out.
Considering the pain I had with spring dependencies in the tools module I can understand the problem.

Reply:
> I think I can remove the scope, but that would make it the default scope compile wouldn't it?

It would, but the idea is that wherever we actually use the dependency, we declare the scope (typically test, though as you found in the compliance suites, sometimes compile). So the root pom defines the version of the library, but not the scope.